dataland-e2etests/src/test/kotlin/org/dataland/e2etests/Constants.kt

```diff
@@ -27,6 +28,7 @@ const val REVIEWER_USER_ID = "f7a02ff1-0dab-4e10-a908-7d775c1014ae"
 val REVIEWER_USER_PASSWORD: String = System.getenv("KEYCLOAK_REVIEWER_PASSWORD")
 val REVIEWER_EXTENDED_ROLES = listOf("ROLE_REVIEWER")
 
+
```

Comment:
Why an additional empty line? Just as a reminder: an analog block for the data_premium_user is neeeded here to make the tests run

Reply:
adapted the file